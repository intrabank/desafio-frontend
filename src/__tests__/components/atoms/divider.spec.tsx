import '@testing-library/jest-dom';
import Divider from '../../../components/atoms/divider';
import renderer from 'react-test-renderer';

describe('Divider', () => {
	it('should render card component', () => {
		const tree = renderer.create(<Divider />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
