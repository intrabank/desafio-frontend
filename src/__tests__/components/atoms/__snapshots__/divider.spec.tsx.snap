// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render card component 1`] = `
<div
  className="c-UcqsO"
/>
`;
